### PR TITLE
Feature/df 359 jobad details sven

### PR DESCRIPTION
**Pull request by 99999989**

### CODE REVIEW

File: alv-portal-ui/src/app/job-ad-search/job-search/filter-panel/filter-panel.component.html [hunk 1]
```diff
@@ -1,55 +1,64 @@
...
+  {{'job-search.filter.title' | translate}}
+</button>
+<div class="title-bar alv-bar text-truncate mb-3">
+  {{'job-search.filter.title' | translate}}
+</div>
+
+<form class="filter-form pb-2"
+      [class.expanded]="expanded">
+  <alv-select
+    *alvHasAnyAuthority="['ROLE_JOBSEEKER_CLIENT']"
```
Comment (pavax) [inline alv-portal-ui/src/app/job-ad-search/job-search/filter-panel/filter-panel.component.html]: doesn't work now anymore

File: alv-portal-ui/src/app/job-ad-search/job-search/filter-panel/filter-panel.component.html [hunk 2]
```diff
@@ -1,55 +1,64 @@
...
-<form formGroupName="form">
+<button class="title-bar alv-bar text-truncate mb-3 w-100"
+        (click)="expanded = !expanded">
+  {{'job-search.filter.title' | translate}}
+</button>
+<div class="title-bar alv-bar text-truncate mb-3">
+  {{'job-search.filter.title' | translate}}
+</div>
+
+<form class="filter-form pb-2"
```
Comment (pavax) [inline alv-portal-ui/src/app/job-ad-search/job-search/filter-panel/filter-panel.component.html]: why can't we use a formGroup

File: alv-portal-ui/src/app/job-ad-search/job-search/result-list-item/result-list-item.component.html [hunk 3]
```diff
@@ -1,30 +1,34 @@
...
-        {{badge.label|translate}}
+        {{badge.label | translate}}
       </ng-template>
     </span>
-  </div>
 
-  <div class="line-clamp"
-       [innerHtml]="result.description | markdownEscape | safeHtml"></div>
+  <div class="line-clamp-2"
+       [style.-webkit-box-orient]="'vertical'"
```
Comment (pavax) [inline alv-portal-ui/src/app/job-ad-search/job-search/result-list-item/result-list-item.component.html]: does it work for edge and firefox?

File: alv-portal-ui/src/assets/i18n/__parsed_extra.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (pavax) [inline alv-portal-ui/src/assets/i18n/__parsed_extra.json]: remove

File: alv-portal-ui/src/app/job-ad-search/job-search/filter-panel/filter-panel.component.html (hunk 2 above)
Comment (pavax) [reply alv-portal-ui/src/app/job-ad-search/job-search/filter-panel/filter-panel.component.html]: done

File: alv-portal-ui/src/app/job-ad-search/job-search/filter-panel/filter-panel.component.html (hunk 1 above)
Comment (99999989) [reply alv-portal-ui/src/app/job-ad-search/job-search/filter-panel/filter-panel.component.html]: works again

File: alv-portal-ui/src/assets/i18n/__parsed_extra.json (hunk 4 above)
Comment (99999989) [reply alv-portal-ui/src/assets/i18n/__parsed_extra.json]: You will do it

**Pull request merged**